### PR TITLE
[FCE-578] Update metadata and fix iOS metadata handling

### PR DESCRIPTION
**Pull request by maksg**
## Description

Updated metadata to match Fishjam and removed `join` method on iOS SDK to match Android while also passing correct peer metadata instead of empty one.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by MiloszFilimowski — COMMENTED** [review]
No changes required on Android? 🤔

File: packages/react-native-client/ios/RNFishjamClient.swift [hunk 1]
```diff
@@ -402,11 +394,11 @@ class RNFishjamClient: FishjamClientListener {
     func getPeers() throws -> [[String: Any?]] {
         let endpoints = RNFishjamClient.getLocalAndRemoteEndpoints()
         return endpoints.compactMap { endpoint in
-            [
+            return [
```
Comment (karkakol) [inline packages/react-native-client/ios/RNFishjamClient.swift:397]: **question:** 
Is return statement needed here?

File: packages/ios-client/Sources/FishjamClient/FishjamClientInternal.swift [hunk 2]
```diff
@@ -80,11 +80,11 @@ internal class FishjamClientInternal: WebSocketDelegate, PeerConnectionListener,
...
     }
 
-    func join(peerMetadata: Metadata = Metadata()) {
+    func join() {
         commandsQueue.addCommand(
             Command(commandName: .JOIN, clientStateAfterCommand: .JOINED) {
-                self.localEndpoint = self.localEndpoint.copyWith(metadata: peerMetadata)
-                self.rtcEngineCommunication.connect(metadata: peerMetadata)
+                self.localEndpoint = self.localEndpoint.copyWith(metadata: self.config?.peerMetadata)
+                self.rtcEngineCommunication.connect(metadata: self.config?.peerMetadata ?? Metadata())
```
Comment (karkakol) [inline packages/ios-client/Sources/FishjamClient/FishjamClientInternal.swift:87]: **suggestion (non-blocking):** 
```suggestion
                self.rtcEngineCommunication.connect(metadata: self.localEndpoint.metadata)
```

File: packages/react-native-client/ios/RNFishjamClient.swift (hunk 1 above)
Comment (maksg) [reply packages/react-native-client/ios/RNFishjamClient.swift:397]: I forgot to remove that after testing values in this map. Thanks!

File: examples/fishjam-chat/screens/ConnectWithRoomManagerScreen.tsx [hunk 3]
```diff
@@ -52,12 +52,12 @@ async function getFishjamServer(
   }
   const tokenData = (await response.json()) as {
     url: string;
-    token: string;
+    participantToken: string;
```
[the remark this answers is not in the document]
Comment (maksg) [reply examples/fishjam-chat/screens/ConnectWithRoomManagerScreen.tsx:55]: It's not merged yet in API but I can change that so it works in the future

**Review by karkakol — APPROVED** [review]
👍

**Pull request merged**